frontend/src/lib/components/ProductEmptyState/ProductEmptyState.tsx

```diff
@@ -0,0 +1,72 @@
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { IconOpenInNew, IconPlus } from 'lib/lemon-ui/icons'
+import { BuilderHog3, DetectiveHog } from '../hedgehogs'
+
+export const ProductEmptyState = ({
+    productName,
+    thingName,
+    description,
+    action,
+    actionElementOverride,
+    docsURL,
+    customHog,
+}: {
+    // The name of the product, e.g. "Cohorts"
```

Comment:
These are all plain single line comments, but if you use the JSDoc syntax 
```suggestion
    /** The name of the product, e.g. "Cohorts" */
```
(note the two asterisks at the beginning), code editors will pick this description up for hover tooltips (since this syntax specifically means "I am describing the thing below for documentation purposes")